docs/system_overview/configuration.md

```diff
@@ -82,132 +128,126 @@ This file contains the following top level entries:
 
   - `pam_password_min_time` (optional) - Minimum allowed PAM password lifetime, in seconds
 
-  - `re_data_variable_mapping_set` (required) - An array of file names comprising the list of data to variable mappings used by the rule engine, for example: [ "core" ] which references 'core.dvm'
+  - `plugin_configuration` (required) - Holds several different properties such as database connection information, rule engine configuration, etc.
 
-  - `re_function_name_mapping_set` (required) - An array of file names comprising the list of function name map used by the rule engine, for example: [ "core" ] which references 'core.fnm'
+      - `authentication` (optional) -
 
-  - `re_rulebase_set` (required) - An array of file names comprising the list of rule files used by the rule engine, for example: [ "core" ] which references 'core.re'.  This array is ordered as the order of the rule files affects which (multiply) matching rule would fire first.
```

Comment:
Notice I removed the `re_*` properties.

These look like they are from a legacy version of iRODS. Aren't they part of the NREP now?

Reply:
yes, they are now plugin_specific_configuration....